src/main/java/org/broadinstitute/hellbender/engine/dataflow/DataflowCommandLineProgram.java

```diff
@@ -41,9 +43,20 @@ public String getHelpDoc(){
         }
 
     }
-    @Argument(fullName="runner", doc="What type of pipeline runner to use for dataflow.  Any type other than LOCAL requires that project and staging be set.")
+    @Argument(fullName="runner", doc="What type of pipeline runner to use for dataflow.  " +
+        "BLOCKING or NONBLOCKING requires that project and staging be set.")
     protected PipelineRunnerType runnerType = PipelineRunnerType.LOCAL;
 
+    @VisibleForTesting
+    public static String getRunnerTypeName(String runnerSimpleName) {
```

Comment:
Why have this lookup from class simple name -> enum name when we just specify the enum name in the system property? 

Needs a javadoc explaining what this is for.